src/requirements-windows.txt

```diff
@@ -0,0 +1,6 @@
+playsound==1.2.2
+pytest==5.0.1
+applicationinsights==0.11.9
+python-socketio==4.3.1
+requests==2.22.0
+pywin32>=224
```

Comment:
I am not super familiar of what is the best practice for dealing with python dependencies, but it seems like we can do some kind of conditional dependencies for requirements.txt, [check reference](https://stackoverflow.com/a/54281345). Maybe we can try something like that instead?
It seems easier than having to maintain two different requirements files (+ we might want to add something later on for Mac?).

Reply:
Added the conditional dependency! Nice catch :)